EvidenceApi.Tests/V1/Gateways/EvidenceGatewayTests.cs

```diff
@@ -383,7 +376,7 @@ public void CanGetEvidenceRequestsByTeamAndResidentReferenceId()
         }
 
         [Test]
-        public void CanGetEvidenceRequestsWithDocumentSubmissions()
+        public void CanGetEvidenceRequestsWithDocumentSubmissionsInExpectedOrder()
```

Comment:
What is the expected order? We should be more explicit that we're talking about `descending` order